components/BottomSheet/examples/BottomSheetPresentationExample.m

```diff
@@ -0,0 +1,87 @@
+/*
+ Copyright 2017-present the Material Components for iOS authors. All Rights Reserved.
```

Comment:
please use the new style of copyright (// vs /*), and it should also be 2018. please see other example files for reference

Reply:
Done.